readme.md

```diff
@@ -12,10 +12,18 @@ delete specific information. See the image below to get a better understanding.
 ## End Goal
 ![Project Bot End Result](Images/slash.png)
 
+
+
+### Knex Configuration:
+In order for your Knex migrations to function correctly be sure to setup your
+local project .env file with your own personal host, user, password and database name
```

Comment:
Maybe put `.env` in back ticks (`). Same with the example and .gitignore.

Reply:
Done.